README.md

```diff
@@ -20,7 +20,7 @@ intelligible and resemble the speakers who recorded the source material.
 
 Initially, RHVoice could speak only Russian. Now it also supports:
 
-* American English
+* American and Scottish English
```

Comment:
not sure about that one.

Reply:
Based on https://rhvoice.org/en-voices/#scottish-english